pkgs/by-name/lu/lux-cli/package.nix

```diff
@@ -0,0 +1,111 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  libgit2,
+  gnupg,
+  libgpg-error,
+  gpgme,
+  installShellFiles,
+  luajit,
+  makeWrapper,
+  symlinkJoin,
+  lua51Packages,
+  lua52Packages,
+  lua53Packages,
+  lua54Packages,
+  nix,
+  versionCheckHook,
+}:
+let
+  lux-lua-bundle = symlinkJoin {
+    name = "lux-lua-bundle";
+    paths = [
+      lua51Packages.lux-lua
+      lua52Packages.lux-lua
+      lua53Packages.lux-lua
+      lua54Packages.lux-lua
+    ];
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "lux-cli";
+
+  version = "0.3.13";
```

Comment:
Anything important in new release that we want to bump right away? https://github.com/nvim-neorocks/lux/compare/v0.3.13...v0.3.14

Reply:
Oh yeah, we probably want that. Weird, I thought I had packaged the latest release